frontend/packages/dev-console/src/components/pipelines/modals/common/SecretAnnotation.tsx

```diff
@@ -11,8 +12,9 @@ type SecretAnnotationParam = {
 const SecretAnnotation: React.FC<SecretAnnotationParam> = (props) => {
   const { fieldName, isReadOnly = false } = props;
   return (
-    <div className="row">
-      <div className="col-lg-6">
+    <>
+      <p className="odc-secret-annotation__label">Designate provider to be authenticated</p>
```

Comment:
I think we probably want to support BEM here.

```suggestion
    <div className="odc-secret-annotation">
      <p className="odc-secret-annotation__label">Designate provider to be authenticated</p>
```